config/routes.rb

```diff
@@ -167,7 +167,7 @@
     end
 
     namespace :migration, constraints: ->(_request) { Rails.application.config.npq_separation[:migration_enabled] } do
-      resources :migrations, only: %i[index]
+      resources :migrations, only: %i[index create]
```

Comment:
I was wondering if we should have a tab appear for super user admins as in `npq-separation/admin/migration/migrations` for this or leave out of admin?

Reply:
I would be inclined to leave it off in to avoid any accidental usage/clicking, but happy to add it in if you feel differently?

Reply:
makes sense, let's leave out 👍